### DISCUSSION TITLE
Some another nice extension to change UI  - nevysha-ui

### DISCUSSION THREAD

**Discussion by Centurion-Rome** [vladmandic/sdnext#703] [General]
Just saw it on reddit. Maybe someone will need it?
https://github.com/Nevysha/a1111-nevysha-comfy-ui
![at5mml7nidxa1](https://user-images.githubusercontent.com/75789136/235730017-2238c7d7-ee5b-432f-934e-3b1db9dc2ce7.png)

org reddit: https://www.reddit.com/r/StableDiffusion/comments/135czga/auto1111_ui_improvement_extension/

**Comment by Nacurutu** [comment]
Looks cool, its based on anapnoes fork... however, anapnoe and vladmantic are working together to adapt in a near future Anapoe ui/ux with this fork..

**Reply by iDeNoh** [reply]
That's fantastic news, I hope they give mobile users some love, the current layout on mobile is... Suboptimal at best.

**Reply by vladmandic** [reply]
now you are being kind :)
suboptimal? i'd rather say non-existent...
